.github/workflows/Tests.yml

```diff
@@ -0,0 +1,75 @@
+name: Tests
+
+on:
+  pull_request:
+    branches:
+      - "**"
+  push:
+    branches:
+      - "latest"
```

Comment:
Maybe could be removed...

Reply:
This is done to not have two tests runs on every commit.